CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```

Comment:
this line needs a change, 

Contributor Mission Support System Code of Conduct